src/task.cc

```diff
@@ -46,7 +46,7 @@ namespace detail {
 			subrange<3> req;
 			switch(kernel_dims) {
 			case 0:
-				[[fallthrough]]; // cl::sycl::range is not defined for the 0d case, but since only constant range mappers are useful in the 0d-kernel case
+				[[fallthrough]]; // celerity::range is not defined for the 0d case, but since only constant range mappers are useful in the 0d-kernel case
```

Comment:
Outdated comment

Reply:
I've opted to get rid of this fall-through and explicitly implement the path for RMs on 0-dimensional buffers or tasks. This a slight breaking change as it disallows `one_to_one` for accessing 1-dimensional buffers in master-node host tasks (which are 0-dimensional). This was used by one test, but IMO it was wrong to support this in the first place.